pybaseball/playerid_lookup.py

```diff
@@ -52,7 +52,22 @@ def get_lookup_table(save=False):
     table['name_first'] = table['name_first'].str.lower()
     return table
 
+def name_similarity(last: str, first: str, player_table: pd.DataFrame) -> pd.DataFrame:
+    """Calculates similarity of first and last name provided with all players in player_table
 
+    Args:
+        last (str): Provided last name
+        first (str): Provided first name
+        player_table (pd.DataFrame): Chadwich player table including names
+
+    Returns:
+        pd.DataFrame: player table with levenshtein distance added
+    """
+    player_table["lastname_dist"] = player_table.dropna().apply(lambda row : levenshtein(row["name_last"], last),axis=1)
```

Comment:
this adds new rows to the `player_table` but it'd be better if this function doesn;t mutate the data

Reply:
fixed in commit e0fe3d0